tensorflow/core/util/mkl_types.h

```diff
@@ -118,7 +121,8 @@ namespace tensorflow {
 #define ADD_MD add_pd
 #define ALGORITHM mkldnn
 #define ALGORITHM_UNDEF ALGORITHM::algorithm_undef
-#define CPU_STREAM(engine) stream(stream::kind::eager_nostore)
+#define BN_FLAGS mkldnn
+#define CPU_STREAM(engine) stream(stream::kind::eager)
```

Comment:
According to PR https://github.com/tensorflow/tensorflow/pull/37001, this should be `eager_nostore` to prevent memory leaks.
```suggestion
#define CPU_STREAM(engine) stream(stream::kind::eager_nostore)
```

Reply:
There is some issue here. Locally it was no_store. Let me check this